controllers/series.js

```diff
@@ -0,0 +1,55 @@
+const models = require('../models')
+
+const getSeriesByName = async (request, response) => {
+  try {
+    const { name } = request.params
+
+    const series = await models.Series.findAll({
+      where: {
+        name: { [models.Op.like]: `%${name}%` },
+      },
+      include: {
+        model: models.Actors
+      }
+    })
+
+    return series.length > 0
+      ? response.send(series)
+      : response.status(404).send('No series found by that name, please try again')
+  } catch (error) {
+    return response.status(500).send('Unable to retrieve series, please try again')
+  }
+}
+
+const saveNewSeries = async (request, response) => {
+  try {
+    const { name, airDates, numberOfSeasons } = request.body
```

Comment:
Your docs say `{ "series": "New Series", "airdates": "start year - end year", "seasons": 1 }` so this route doesn't work.